test/input.txt

```diff
@@ -86,6 +86,13 @@
   viewall 4
   viewall 5
 
+##########################################################
+# test sort persons command
+##########################################################
+
+  # should show persons in alphabetical order
+  sort
```

Comment:
Since the address book is already in alphabetical order, how does this test verify that the program is not just outputting the address book directly.